src/zen/workspaces/zen-workspaces.css

```diff
@@ -6,6 +6,16 @@
 @namespace html 'http://www.w3.org/1999/xhtml';
 @namespace xul 'http://www.mozilla.org/keymaster/gatekeeper/there.is.only.xul';
 
+.zen-group-loading-icon {
```

Comment:
Instead of a loading icon, shouldn't we make the separator a squiggly line going up and down? Like arc.